seeds/001-4:57-1:16:19-cohort-seeds.js

```diff
@@ -0,0 +1,22 @@
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('cohorts').truncate()
+    .then(function () {
+      // Inserts seed entries
+      return knex('cohorts').insert([
+        {name: 'rowValue1'},
+        {name: 'rowValue2'},
+        {name: 'rowValue3'},
+        {name: 'rowValue4'},
+        {name: 'rowValue5'},
+        {name: 'rowValue6'},
+        {name: 'rowValue7'},
+        {name: 'rowValue8'},
+        {name: 'rowValue9'},
+        {name: 'rowValue10'},
+        {name: 'rowValue11'},
+        {name: 'rowValue12'}
```

Comment:
This might be better done iteratively so that it's easily expandable later without much work on your part:
```javascript
const SIZE = 12;
return knex('cohorts').insert(
  new Array(SIZE).fill(null).map((_, i) => ({
    name: `rowValue${i+1}`
  }))
);
```